rswag-specs/lib/rswag/specs/request_factory.rb

```diff
@@ -101,8 +101,27 @@ def add_verb(request, metadata)
         request[:verb] = metadata[:operation][:verb]
       end
 
+      def base_path_from_servers(swagger_doc, use_server = :default)
+        return '' if swagger_doc[:servers].nil? || swagger_doc[:servers].empty?
+        base_paths = swagger_doc[:servers].map do |server|
+          variables = {}
+          server.fetch(:variables, {}).each_pair { |k,v| variables[k] = v[use_server] }
+          server[:url].gsub(/\{(.*?)\}/) { variables[$1.to_sym] }
+        end
+        URI(base_paths.first).path
```

Comment:
why map over all the servers if only the first one is being consumed? :think

Reply:
You're right. This was a result of my confusion with supporting multiple servers and it is not necessary.